WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js

```diff
@@ -453,7 +487,13 @@ function() {
  */
 ZmBriefcaseItem.createFromDom =
 function(node, args) {
-	var item = new ZmBriefcaseItem(node.id, args.list);
+	var id;
+	if (node.sfid && node.l == ZmFolder.ID_FILE_SHARED_WITH_ME) {
+	    id = node.sfid;
```

Comment:
mix of spaces and tabs

Reply:
As per the standard I have done using spaces. But earlier indentation  are in tabs.  Let me know which one should we keep.

Reply:
which standard are you referring to?

Reply:
http://wiki.eng.zimbra.com/index.php/Zimbra_Client_Coding_Conventions#Indentation  here it says we can use either tab or 4 spaces for indentation. When I format code blocks in VS code it  used 4 spaces this caused the mixing of tabs and spaces. converted it back to tabs.